molecule_common/origin-latest/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openshift/origin:latest
+
+RUN yum makecache fast && yum update -y && \
+    yum install -y python sudo yum-plugin-ovl && \
+    sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf
```

Comment:
Same as with the other Dockerfile:

- python is already in the base image;
- `'s/^plugins=0$/plugins=1/g'`
- missing a `yum clean all`?